src/light/bake.js

```diff
@@ -3,34 +3,39 @@ import callServices from '../utils/services/call.js'
 import whenDone from '../utils/processing/when-done.js'
 
 // main
+function bakeStage(stage) {
+  return function bake(storageId, options) {
+    // API
+    options = options || {}
 
-function bake(storageId, options) {
-  // API
-  options = options || {}
-  var sunDirection = sunDirection || [
-    0.7487416646324341,
-    -0.47789104947352223,
-    -0.45935396425474223
-  ]
+    // Optional bake parameters for API call
+    var bakeSettings = { sunDirection: options.sunDirection || [ 0.75, -0.48, -0.46 ] }
+    if (options.lightMapCount) { bakeSettings.lightMapCount = options.lightMapCount }
```

Comment:
this has to be either (prefered option in one line without semicolon must not have curly brackets)
```
if (options.lightMapCount) bakeSettings.lightMapCount = options.lightMapCount
```
or (without semicolon with curly brackets: use multiple lines)
```
if (options.lightMapCount) {
  bakeSettings.lightMapCount = options.lightMapCount
}
```
or (rather not common: in one line with curly brackets: requires semicolon)
```
if (options.lightMapCount) { bakeSettings.lightMapCount = options.lightMapCount; }
```

Reply:
thanks!